packages/api/src/__tests__/e2e/mapi/consolidated/consolidated.ts

```diff
@@ -87,3 +95,29 @@ export function makeAllergyIntollerance({
     recordedDate: "2010-09-03T03:10:10-05:00",
   };
 }
+
+export function checkConsolidatedHtml({
+  html,
+  patientId,
+  outputExpectedFileName = "consolidated-expected",
+  outputReceivedFileName = "consolidated-received",
+}: {
+  html: string;
+  patientId: string;
+  outputExpectedFileName?: string;
+  outputReceivedFileName?: string;
+}): boolean {
+  const date = dayjs().utc().format("YYYY-MM-DD");
+
+  const consolidatedHtmlFile = fs.readFileSync(`${__dirname}/consolidated-template.html`, "utf8");
+  const compiled = template(consolidatedHtmlFile);
+  const expectedContent = compiled({ date, patientId }).toString().trim();
+  const receivedContent = html.trim();
+
+  const isMatch = receivedContent === expectedContent;
+  if (!isMatch) {
+    fs.writeFileSync(`${e2eResultsFolderName}/${outputExpectedFileName}.html`, expectedContent);
+    fs.writeFileSync(`${e2eResultsFolderName}/${outputReceivedFileName}.html`, receivedContent);
+  }
```

Comment:
I am a little confused by writing things to files in e2e testing. This is running in cicd right? what file system are they getting saved to when run there and how could we look at them? Or is only for when running e2e tests local?

Reply:
I see now the way artifacts are being using in the _e2e-tests.yml file. This is a cool new pattern in our workflow files that you should
a. communicate to team when time is appropriate
b. because it is new maybe explain a little more in PR description or in a comment somewhere because it wasn't intuitive and I had to do a little digging to understand where the files were going

Reply:
This is not a new pattern, we have this for a long time on internal. I agree we don't use it often.

I'll add a note to the PR description and a message on engineering.

Reply:
Ah alright. I only looked on oss and didn't see the pattern so assumed it was new. Sounds good to me!